config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
```

Comment:
- [ ] The routes are not present correcltu. Please make sure you have followed the correct rotues so it leads to correct page.

Reply:
The app works correctly and the routes are well defined: each transaction belongs to one category